CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v0.13.0
+
+The primary motivation for this release is to add support for fetching images on the Docker Registry 2.0. It also includes other small improvments.
```

Comment:
It also includes TPM support to provide a tamper-proof audit log of every started pod and other small improvements.